### PR TITLE
add maven-failsafe-plugin version definition #209

### PR DESCRIPTION
**Pull request by btokadanob**
- [x] add vesrion 2.19 maven-failsafe-plugin

---

#209
please review
by btokadanob

---

### CODE REVIEW

File: projectName-selenium/pom.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
+                <version>${maven.failsafe.plugin.version}</version>
```
Comment (ikeyat) [inline projectName-selenium/pom.xml]: - [ ] Please add ``<pluginManagement>`` at parent pom and define version there.

**Comment by btokadanob** [comment]
- [ ] fix definition position

---

f2a618c77fb838b394fd4a4f28d8deea2129a3e9
629a5e7e56a277f1a0dd6b06fd8d397a8770638a

---

File: pom.xml [hunk 2]
```diff
@@ -134,5 +146,5 @@
         <org.codehaus.mojo.build-helper-maven-plugi.version>1.9.1</org.codehaus.mojo.build-helper-maven-plugi.version>
         <postgresql.version>9.3-1102-jdbc41</postgresql.version>
         <ojdbc.version>12.1.0.2</ojdbc.version>
-    </properties>
-</project>
\ No newline at end of file
+        <maven.failsafe.plugin.version>2.19</maven.failsafe.plugin.version>
```
Comment (btshimizukza) [inline pom.xml]: - [ ] change to `<maven-failsafe-plugin.version>` (`.` -> `-`)
- [ ] move down of `<org.codehaus.mojo.build-helper-maven-plugi.version>`

File: pom.xml [hunk 3]
```diff
@@ -134,5 +146,5 @@
...
+    </properties></project>
```
Comment (btshimizukza) [inline pom.xml]: - [ ] Please add line break after `</properties>`.

**Comment by btokadanob** [comment]
- [ ] fix version properties

---

5094ffce3d61d85b62b0e7b182da089bbe3b172b

---

**Pull request merged**